integration/test_resources/help/policy_disable

```diff
@@ -0,0 +1,32 @@
+Disable Policies by ID or all policies matching a tag.
+
+To disable a single policy by it's ID:
+	lacework policy disable lacework-policy-id
+
+To disable all policies for Aws CIS 1.4.0:
+	lacework policy disable --tag framework:cis-aws-1-4-0
+
+To disable all policies for Gcp CIS 1.3.0:
+	lacework policy disable --tag framework:cis-gcp-1-3-0
```

Comment:
All of this text needs to be updated to render Markdown correctly, you have to add an empty line in between regular text and "code text".

These instructions will look like this in Markdown:

<img width="1724" alt="Screen Shot 2022-04-13 at 3 00 13 PM" src="https://user-images.githubusercontent.com/5712253/163185866-c9fabd94-2fd0-4289-bb6c-52ca32868884.png">

But if you add a carrier return in between, it will show it as code:

<img width="1525" alt="Screen Shot 2022-04-13 at 3 00 32 PM" src="https://user-images.githubusercontent.com/5712253/163186027-297bda54-d5fe-4228-b802-02f144422104.png">

Please update all texts similar to this one.

Reply:
So, this should be looking like this:
```suggestion
To disable a single policy by it's ID:

	lacework policy disable lacework-policy-id

To disable all policies for Aws CIS 1.4.0:

	lacework policy disable --tag framework:cis-aws-1-4-0

To disable all policies for Gcp CIS 1.3.0:

	lacework policy disable --tag framework:cis-gcp-1-3-0
```

Reply:
fixed